src/binaryen-c.h

```diff
@@ -1026,6 +1033,11 @@ BinaryenModuleAllocateAndWriteResult
 BinaryenModuleAllocateAndWrite(BinaryenModuleRef module,
                                const char* sourceMapUrl);
 
+// Serialize a module in s-expression form. Implicity allocates the returned
+// char* with malloc(), and expects the user to free() them manually
+// once not needed anymore.
+char* BinaryenModuleAllocateAndWriteSExpr(BinaryenModuleRef* module);
```

Comment:
How about `Text` instead of `SExpr`, since while the current text format is s-expressions, perhaps this may change in time?